spec/models/user_spec.rb

```diff
@@ -24,4 +24,37 @@
   it { should have_valid(:phone_number).when(1234123123)}
   it { should_not have_valid(:phone_number).when(nil, "", 2331)}
 
+
+  it "Total_vocabulary sould return the total of interactions of user's bots" do
+    user= FactoryGirl.create(:user)
+    bot1 = FactoryGirl.create(:bot, user: user)
+    bot2 = FactoryGirl.create(:bot, user: user)
+    10.times do
+
+      Interaction.create(
+        category: 'Sentence',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.